osclib/core.py

```diff
@@ -288,19 +288,33 @@ def binary_src_debug(binary):
 
 
 @memoize(session=True)
-def devel_project_get(apiurl: str, target_project: str, target_package: str) -> Union[Tuple[str, str], Tuple[Literal[None], Literal[None]]]:
-    """Fetch the devel project & package name of the supplied ``target_project``
-    and ``target_package`` and return the devel project and devel package name
-    as a tuple. If no devel project has been defined, then return ``None,
-    None``.
-
+def git_factory_devel_project_dict_get(apiurl: str) -> Dict[str, str]:
```

Comment:
I don't like the `_dict_get` suffix, the get if at all should be a verb at the beginning. 

how about `git_factory_pkg_project_mapping` ?

Reply:
`factory_git_devel_projects_mapping` ?